src/orchard/misc.clj

```diff
@@ -156,3 +156,17 @@
   "Return true if n is a namespace object"
   [ns]
   (instance? clojure.lang.Namespace ns))
+
+;; Drop this in favor of clojure.core/requiring-resolve at some point?
```

Comment:
I think they made it private.

Reply:
`requring-resolve` is public. They made `serialized-require` private which ensures requiring is done under a lock.